flume/conf/README.md

```diff
@@ -0,0 +1,54 @@
+# Cygnus configuration Procedure
```

Comment:
Please add a comment with "Fixed in <commit number>" in this comment thread. Otherwise, with PRs with many comments, it gets complex to trace changes due to code review.